WallsAndHoles/rendermap.h

```diff
@@ -0,0 +1,15 @@
+#ifndef RENDERMAP_H
```

Comment:
Don't need this to be a class as it doesn't actually store anything. All the methods can just be static.

Moreover, non of this should be stored in it's own file. To me it would make the most sense to put this in MapCell. We will go over design change ideas after today's scrum meeting.